ycsb-mongodb/mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -233,6 +344,47 @@ private static AutoEncryptionSettings generateEncryptionSettings(String url, Boo
         return autoEncryptionSettings;
     }
 
+    private static ArrayList<Long> parseCommaSeparatedIntegers(String toParse, int outputListSize, long defaultValue) {
+        if (toParse.trim().isEmpty()) {
+            return null;
```

Comment:
if you guarantee that you never return null (rather an empty list) then you can drop the checks and the  usage becomes  simpler

```
        for (Long value: parseCommaSeparatedIntegers(cardinalities, -1, 0)) {
            if (value <= 0) {
                continue;
            }

....
```

Reply:
Done. No longer returning null in the helper methods.